pep-0343.txt

```diff
@@ -853,7 +853,6 @@ refers to an action which is to be done in the ``__exit__`` method.
             try:
                 try:
                     for context in contexts:
-                        mgr = context.__context__()
```

Comment:
This leaves mgr undefined, used on the next two lines.

Reply:
Good spot! Fixed.